android-sdk/src/main/java/com/mobilecoin/lib/BlockchainClient.java

```diff
@@ -42,16 +42,23 @@
      * Fetch or return cached current minimal fee
      */
     @NonNull
-    synchronized UnsignedLong getOrFetchMinimumFee() throws NetworkException {
```

Comment:
getOrFetchLastBlockInfo is already synchronized, so we don't actually need this to be